src/index.html

```diff
@@ -8,10 +8,162 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript" src="main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="blocks">
+
+        <legend>Personal information</legend>
+        <div class="inputs">
+          <label>
+          Surname:
+            <input type="text" name="Surname"
+             autocomplete="off">
```

Comment:
```suggestion
            <input
               type="text"
               name="Surname"
               autocomplete="off"
             >
```
follow our [styleguide](https://mate-academy.github.io/style-guides/htmlcss.html)(everywhere) 😉